src/styles/_achievementComments.scss

```diff
@@ -0,0 +1,23 @@
+.mission-comments {
```

Comment:
Can we use a more general term? e.g. assessment-comments. Same for 'mission-button' below